x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/datafeed_step/components/datafeed_preview_flyout/datafeed_preview_flyout.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, FC, useState, useContext, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiFlyout,
+  EuiFlyoutFooter,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiSpacer,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { CombinedJob } from '../../../../../common/job_creator/configs';
+import { MLJobEditor } from '../../../../../../jobs_list/components/ml_job_editor';
+import { JobCreatorContext } from '../../../job_creator_context';
+import { mlJobService } from '../../../../../../../services/job_service';
+import { ML_DATA_PREVIEW_COUNT } from '../../../../../../../../common/util/job_utils';
+
+const EDITOR_HEIGHT = '800px';
+export enum EDITOR_MODE {
+  HIDDEN,
+  READONLY,
+  EDITABLE,
+}
+interface Props {
+  isDisabled: boolean;
+}
+export const DatafeedPreviewFlyout: FC<Props> = ({ isDisabled }) => {
```

Comment:
I'm wondering if we need to make the datafeed preview available on the second step of the wizard too, as the preview changes as you add fields in to detectors?

Reply:
Flyout added to Pick fields and Summary steps in f4f7ba124c9791940d593f6f9e14851e8580e5fc